Source/WebKit/WebProcess/WebPage/dmabuf/AcceleratedSurfaceDMABuf.cpp

```diff
@@ -657,6 +671,14 @@ void AcceleratedSurfaceDMABuf::didRenderFrame(WebCore::Region&& damage)
 
     m_target->didRenderFrame();
     WebProcess::singleton().parentProcessConnection()->send(Messages::AcceleratedBackingStoreDMABuf::Frame(m_target->id(), WTFMove(damage), WTFMove(renderingFence)), m_id);
+    if (m_target)
+        m_target->resetDamage();
```

Comment:
Maybe this could be done automatically in Target::didRenderFrame? and we don't need resetDamage()?